cfgov/unprocessed/css/forms.less

```diff
@@ -664,8 +658,9 @@
 
 .form-l-inset {
     border-right-width: 2.5px;
-    + .form-l-inset { border-left-width: 2.5px; }
-    &:last-child { border-right-width: 15px; }
+    + .form-l_col-1-2.form-l-inset:nth-child(odd) { border-left-width: 2.5px; }
```

Comment:
It renders differently on different browsers, which might make it worth looking into on in another PR. I don't see anything that says that this renders differently than margins, but I'm not the biggest fan of using borders like this.